rllib/agents/ppo/ppo.py

```diff
@@ -462,3 +477,54 @@ def execution_plan(
         return StandardMetricsReporting(train_op, workers, config).for_each(
             lambda result: warn_about_bad_reward_scales(config, result)
         )
+
+    @ExperimentalAPI
+    def training_iteration(self) -> ResultDict:
+        # Collect SampleBatches from sample workers
+        rollouts = synchronous_parallel_sample(self.workers)
+
+        # Concatenate the SampleBatches from each worker into one large SampleBatch
+        rollouts = SampleBatch.concat_samples(rollouts)
```

Comment:
We might need some `train_batch_size` check here (while loop just like in rllib/agents/pg/pg.py).
Otherwise, the train batch size might be too small after just a single round of parallel `RolloutWorker.sample()` collection.

Reply:
make it into a util to collect exactly train_batch # of samples from available workers in truncated_episode mode, and at least train_batch # of samples in complete_episode mode?